pkg/manager/member/pd_member_manager.go

```diff
@@ -761,7 +763,11 @@ func getNewPDSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap) (
 	pdContainer.Env = util.AppendEnv(env, basePDSpec.Env())
 	pdContainer.EnvFrom = basePDSpec.EnvFrom()
 	podSpec.Volumes = append(vols, basePDSpec.AdditionalVolumes()...)
-	podSpec.Containers = append([]corev1.Container{pdContainer}, basePDSpec.AdditionalContainers()...)
+	podSpec.Containers, err = MergePatchContainers([]corev1.Container{pdContainer}, basePDSpec.AdditionalContainers())
+	if err != nil {
+		return nil, errors2.Wrap(err, "failed to merge containers spec")
```

Comment:
Why introduce the errors2 here? Can we just use the same error-generating code as the other ones in the same file to keep the same style?